UI/Pages/ContactUs.cshtml

```diff
@@ -9,25 +9,24 @@
         <h1 class="govuk-heading-l" data-testid="contact-us-header">
             Contact us
         </h1>
+        <h2 class="govuk-heading-m" data-testid="contact-us-telephone-header">Telephone</h2>
+        <ul class="govuk-list">
+            <li data-testid="contact-us-telephone">Phone: 0300 373 0891</li>
+            <li>Monday to Friday 8.30am to 5pm, except bank holidays</li>
+        </ul>
         <p class="govuk-body">
-            To give feedback on how our service worked, you can fill in our <a href="https://forms.gle/44KfQyg1YUidrDCi7" data-testid="feedback-form-link"
-                                                                               class="govuk-link" target="_blank">feedback form.</a>
-        </p>
-        <h2 class="govuk-heading-m">For support about tuition partners </h2>
-        <p class="govuk-body">
-            If you have any questions or need support on our service, you can:
-            <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
-                <li>
-                    email <a href="mailto:tutoring.support@service.education.gov.uk" data-testid="mailto-ntp-link"
-                             class="govuk-link">tutoring.support@service.education.gov.uk.</a>
-                </li>
-                <li>call 0300 373 0891 - open Monday to Friday 8.30am to 5pm (except bank holidays)</li>
-            </ul>
-        </p>
-        <h2 class="govuk-heading-m">For issues with your tuition partner </h2>
-        <p class="govuk-body">
-            For help if you have an issue with your tuition partner, you can  <a asp-page="@nameof(ReportIssues)" class="govuk-footer__link" data-testid="report-issues-link">read our guidance.</a>                                                                            
+            <a data-testid="find-out-call-charges-link" class="govuk-link" target="_blank" href="https://www.gov.uk/call-charges">Find out about call charges</a>
```

Comment:
We should open this in the same tab for accessibility as per https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab

```suggestion
            <a data-testid="find-out-call-charges-link" class="govuk-link" href="https://www.gov.uk/call-charges">Find out about call charges</a>
```

Reply:
Okay, I have now made the suggested changes.